qcodes/dataset/data_set.py

```diff
@@ -1807,6 +1846,8 @@ def new_data_set(name: str,
         specs: list of parameters to create this dataset with
         values: the values to associate with the parameters
         metadata: the metadata to associate with the dataset
+        in_memory_cache: Should measured data be keep in memory
+            and available as part of the `dataset.cache` object.
```

Comment:
```suggestion
        in_memory_cache: Should measured data be kept in memory
            and available as part of the `dataset.cache` object, as opposed
            to being persisted
```

Reply:
No not as opposed in addition

Reply:
aaaaaaaaa........ so stupid of me - so in this PR the cache is maintained in memory and we don't yet "stop writing to the DB", right?

Reply:
perfecet!